### PR TITLE
New Icon: Netgear

### LINKED ISSUE #8239: Icon request: Netgear
### Brand Name

NETGEAR

### Website

https://www.netgear.com/

### Popularity Metric

https://www.similarweb.com/website/netgear.com/#overview

image

### Official Resources for Icon and Color

None

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by ripzerotwo**
**Issue:** closes #8239 

**Similarweb rank:** 12,427 (https://www.similarweb.com/website/netgear.com/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
So there is no easily obtainable svg logo so I made the svg in the likeness of the flavicon. I followed the netgear branding guideline (https://www.powershift.netgear.de/images/powershift/collateral/logos-visio-icons/brandguideline_partner_2018.pdf) for the requisite font. There is a discussion to be had on whether the color should be inverted i.e. the N should be white and not transparent. 

netgear

### CODE REVIEW

**Comment by LitoMore** [comment]
According to their guidelines, I'd prefer the complete trademark `NETGEAR`.

**Comment by adamrusted** [comment]
I agree with @LitoMore - as I started looking at this, I realised I've never seen anything other than the full wordmark on their products. The only place the `N` is visible on it's own (At least to my eyes) is the favicon.

File: icons/netgear.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Netgear</title><path d="M0 0v24h24V0H0Zm18.42 18.37h-3.87l-5.68-8.33h-.06l.08 8.33H5.58V5.63h3.89l5.66 8.32h.06l-.08-8.32h3.31v12.74Z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/netgear.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Netgear</title><path d="M23.727 11.375c0-.695-.515-1.002-1.192-1.002h-1.389v3.255h.892v-1.252h.063l.814 1.252H24l-1.023-1.334c.487-.082.75-.47.75-.92m-1.46.458h-.23v-.811h.256c.293 0 .542.051.542.406 0 .354-.275.405-.568.405M18.357 10.373l-1.2 3.256h.923l.196-.6h1.192l.196.6h.921l-1.198-3.256m-.9 2.009.372-1.137h.025l.372 1.137h-.768zM14.446 13.628h2.175v-.716h-1.282v-.562h1.228v-.716h-1.228v-.544h1.282v-.717h-2.175M11.917 12.475h.702c-.036.376-.306.566-.693.566-.588 0-.88-.514-.88-1.019 0-.496.252-1.027.84-1.027.36 0 .62.208.742.518l.847-.336c-.296-.61-.842-.913-1.548-.913-1.107 0-1.81.697-1.81 1.754 0 1.023.725 1.718 1.804 1.718.575 0 1.054-.193 1.39-.639.306-.406.341-.81.35-1.286h-1.744M7.111 11.09h.864v2.539h.906V11.09h.864v-.717H7.111M4.327 13.628h2.175v-.716H5.22v-.562h1.228v-.716H5.22v-.544h1.282v-.717H4.327M2.379 12.363h-.01l-1.477-1.99H0v3.256h.892v-1.995h.01l1.477 1.995h.892v-3.256h-.892"/></svg>
```
Use the complete wordmark

**Comment by PratyushJayachandran** [comment]
@simple-icons/icon-review please review this PR. I took it up and made the requested changes.

File: icons/netgear.svg
Comment (adamrusted) [inline icons/netgear.svg:1]: Unless their brand guidelines state that we _can_ remove the ®️ symbol - we need to keep it.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8935,6 +8935,12 @@
             "source": "https://brand.netflix.com/en/assets/brand-symbol",
             "guidelines": "https://brand.netflix.com/en/assets/brand-symbol"
         },
+        {
+            "title": "Netgear",
+            "hex": "000000",
+            "source": "https://www.powershift.netgear.de/images/powershift/collateral/logos-visio-icons/brandguideline_partner_2018.pdf",
+            "guidelines": "https://www.netgear.com/about/trademarks/"
```
Comment (adamrusted) [inline _data/simple-icons.json:8942]:
```suggestion
            "guidelines": "https://www.netgear.com/about/trademarks"
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Couple of changes listed below - if you wouldn't mind taking a look @PratyushJayachandran

File: _data/simple-icons.json [hunk 3]
```diff
@@ -8935,6 +8935,12 @@
             "source": "https://brand.netflix.com/en/assets/brand-symbol",
             "guidelines": "https://brand.netflix.com/en/assets/brand-symbol"
         },
+        {
+            "title": "Netgear",
```
Comment (adamrusted) [inline _data/simple-icons.json:8939]:
```suggestion
            "title": "NETGEAR",
```
They always capitalize the name.

**Comment by adamrusted** [comment]
image

Updated using icon from right-hand side of Page 3 of PDF in source link. Hex from logo used there.

**Review by PratyushJayachandran — APPROVED** [review]
Perfect match @adamrusted! Thank you for starting the PR @ripzerotwo 🎉

**Pull request merged**